addons/mail/models/res_users.py

```diff
@@ -159,12 +163,12 @@ def systray_get_activities(self):
                     END AS states
                 FROM mail_activity AS act
                 JOIN ir_model AS m ON act.res_model_id = m.id
-                WHERE user_id = %(user_id)s
+                WHERE %(where)s
```

Comment:
It's strange to override the query in such a way. Do you have an example of the override that will be done?

Reply:
Several use cases:

- If the option to archive activities is added, we want to exclude them from this query.
- If you add the activity teams option and want to display all the activities of your team.